Plugins/GitUIPluginInterfaces/IGitModule.cs

```diff
@@ -128,9 +128,23 @@ public interface IGitModule
         string GetEffectiveSetting(string setting);
         string GetEffectiveGitSetting(string setting, bool cache = true);
 
-        /// <summary>Gets the current branch; or "(no branch)" if HEAD is detached.</summary>
+        /// <summary>
+        /// Gets the name of the currently checked out branch.
+        /// </summary>
+        /// <returns>
+        /// The name of the branch (for example: "main"); or "(no branch)", if HEAD is detached.
+        /// </returns>
         string GetSelectedBranch();
 
+        /// <summary>
+        /// Gets the name of the currently checked out branch.
+        /// </summary>
+        /// <param name="setDefaultIfEmpty">Define the value returned if HEAD is detached. <see langword="true"/> to return <see cref="string.Empty"/>; <see langword="false"/> to return "(no branch)".</param>
```

Comment:
```suggestion
        /// <param name="setDefaultIfEmpty">Define the value returned if HEAD is detached. <see langword="false"/> to return <see cref="string.Empty"/>; <see langword="true"/> to return "(no branch)".</param>
```

Reply:
I propose to rename `setDefaultIfEmpty` parameter to `emptyIfDetached` and reword the xmldoc as follows:
```suggestion
        /// <param name="emptyIfDetached">Defines the value returned if HEAD is detached. <see langword="true"/> to return <see cref="string.Empty"/>; <see langword="false"/> to return "(no branch)".</param>
```